frontend/packages/react-app/src/interfaces.ts

```diff
@@ -128,3 +132,12 @@ export interface Target {
   readonly handle: string;
   readonly wallet: string;
 }
+
+export interface TokenListState {
+  isOpen: boolean;
+  tokens: TokenBasic[];
+  tokenAddress: string;
+  inputTokenAddress: string;
+  type: string;
+  color: string;
+}
```

Comment:
IMO, reducer's state interface should be placed on the same file as reducer placed. 
Because state interface is only used for reducers and specific pages and not used for other pages. 
`interfaces.ts` is for more general interfaces.

Reply:
Fixed